e2e/adult_income/serve_client.py

```diff
@@ -46,11 +47,11 @@ def infer(stub, model_name, model_input):
     all_pred = []
     all_target = []
 
-    for (dense, batch_sparse_ids, target) in tqdm(loader, desc="gen batch data..."):
-        batch_data = PersiaBatch()
-        batch_data.add_non_id_type_feature([dense])
-        batch_data.add_id_type_features(batch_sparse_ids, False)
-
+    for (non_id_type_feature, id_type_features, target) in tqdm(
+        loader, desc="gen batch data..."
+    ):
+        batch_data = PersiaBatch(id_type_features, requires_grad=False)
+        batch_data.add_non_id_type_feature(non_id_type_feature)
```

Comment:
The naming of `non_id_type_feature` may cause confusion. For example, we have 5 dense features in adult income dataset, while in our data procession, we concat them and regarded them as 1 dense feature.
How about use `add_concat_non_id_type_features` ?

Reply:
`non_id_type_feature` is a tensor with ndim greater than 1.